test/spec/util/convertBasicSearch.spec.js

```diff
@@ -0,0 +1,98 @@
+(function () {
```

Comment:
For some reason this whole spec was missing, so here it is, with tests for extraneous whitespace.